### ISSUE TITLE
SourceMojo : Use `isEmpty()` to check whether collection is empty or not

### ISSUE THREAD

**Issue by rohanKanojia** [fabric8io/docker-maven-plugin#1746]
### Description

In Java8 we have [List.isEmpty()](https://docs.oracle.com/javase/8/docs/api/java/util/List.html#isEmpty--) method to check whether list size is greater than zero. 
This line
https://github.com/fabric8io/docker-maven-plugin/blob/e8207b98a64c42096fceb1758bd9062d73249d9a/src/main/java/io/fabric8/maven/docker/SourceMojo.java#L81
should be changed to
```java
if (sourceMode == BuildImageSelectMode.first && !imageConfigs.isEmpty()) {
```

## Acceptance Criteria
- [ ] explicit size check is replaced with `isEmpty()` method
- [ ] Project compiles cleanly when `mvn clean install` is run after making changes

**Comment by mdxabu** [comment]
Ok, I'll work on this task. But one problem, the maven dependencies and plugins are not found in Intellij Idea

**Comment by rohanKanojia** [comment]
@mdxabu : You would need to fix your IDE setup if you want to contribute more. 

You would need to do `mvn clean install -DskipTests`and then click on refresh icon `Reload All Maven Projects` in Maven sidebar on top right.

**Comment by mdxabu** [comment]
@rohanKanojia , give me some `good first issues`.

**Comment by rohanKanojia** [comment]
Closed via #1747
